packages/js/email-editor/src/components/styles-sidebar/panels/dimensions-panel.tsx

```diff
@@ -36,19 +40,26 @@ export function DimensionsPanel() {
 					)
 				}
 				label={ __( 'Padding', 'mailpoet' ) }
-				onDeselect={ () =>
+				onDeselect={ () => {
 					updateStyleProp(
 						[ 'spacing', 'padding' ],
 						defaultStyles.spacing.padding
-					)
-				}
+					);
+					recordEvent(
+						'styles_sidebar_screen_layout_dimensions_padding_reset_clicked'
+					);
+				} }
 				className="tools-panel-item-spacing"
 			>
 				<SpacingSizesControl
 					allowReset
 					values={ styles.spacing.padding }
 					onChange={ ( value ) => {
 						updateStyleProp( [ 'spacing', 'padding' ], value );
+						recordEvent(
+							'styles_sidebar_screen_layout_dimensions_padding_updated',
+							{ value }
+						);
```

Comment:
There is a range control and when moving the range control I see multiple events being tracked. I think we could use debounce or something to prevent multiple events triggered while user is adjusting the value and ideally track only the final value or one event to be able to see how many times user changed the setting.

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/6003/commits/15bd07f23341807f82ad61f9246c18d82064fbec